cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/validator/AwsDataAccessRolePermissionValidatorTest.java

```diff
@@ -0,0 +1,82 @@
+package com.sequenceiq.cloudbreak.cloud.aws.validator;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+
+import com.sequenceiq.cloudbreak.cloud.model.filesystem.CloudS3View;
+import com.sequenceiq.common.api.cloudstorage.StorageLocationBase;
+import com.sequenceiq.common.model.CloudIdentityType;
+import com.sequenceiq.common.model.FileSystemType;
+
+public class AwsDataAccessRolePermissionValidatorTest extends AwsIDBrokerMappedRolePermissionValidatorTest {
+    private final AwsDataAccessRolePermissionValidator awsDataAccessRolePermissionValidator =
+        new AwsDataAccessRolePermissionValidator();
+
+    @Override
+    public void testGetUsers() {
+        List<String> expectedUsers = Arrays.asList("hbase", "hdfs", "hive",
```

Comment:
#6700 has been merged, so please free to use `AccountMappingSubject.DATA_ACCESS_USERS` instead.